kuma/users/admin.py

```diff
@@ -60,6 +51,25 @@ def related_user(self, obj):
     related_user.allow_tags = True
     related_user.short_description = 'User account'
 
+    def revisions(self, obj):
+        """HTML link to user's revisions with count"""
+        link = reverse('dashboards.revisions') +\
+                ('?user=%s' % obj.user.username)
+        count = obj.user.wiki_revisions().count()
+        return ('<a href="%(link)s"><strong>%(count)s</strong></a>' %
+                {'link': link, 'count': count})
+    revisions.allow_tags = True
+
+    def websites(self, obj):
+        """The 'website' element in the 'websites' hash from the misc field"""
+        if obj.websites:
+            link = obj.websites.get('website')
+            return ('<a href="%(link)s"><strong>%(link)s</strong></a>' %
+                   {'link': link, 'link': link})
```

Comment:
Ha, named string interpolation where all the names are the same. Will revisit in fixup.